src/measure_theory/interval_integral.lean

```diff
@@ -1256,8 +1256,10 @@ lemma deriv_within_integral_left
   deriv_within (λ u, ∫ x in u..b, f x) s a = -f a :=
 (integral_has_deriv_within_at_left hf hmeas ha).deriv_within hs
 
+/-!
+### Fundamental theorem of calculus, part 2
 
-/-~ ### Theorems pertaining to FTC-2. -/
+This section contains theorems pertaining to FTC-2 for interval integral. -/
```

Comment:
```suggestion
This section contains theorems pertaining to FTC-2 for integrals over intervals. -/
```

Reply:
The language I used is what is used earlier in the document.

Reply:
Oh, OK then. But I think "FTC-2 for interval integral" is still a bit ungrammatical.

How about:
```suggestion
This section contains theorems pertaining to FTC-2 for interval integrals. -/
```
Please also change the "FTC-1 for interval integral" to "FTC-1 for interval integrals" on line 526.

Reply:
Okay